client/src/main.ts

```diff
@@ -176,6 +176,14 @@ async function setupApp(): Promise<void> {
         window.electronAPI.closeApp();
       }
     });
+    window.electronAPI.receive('parsec-force-close-request', async () => {
```

Comment:
You can just pass a `force: boolean = false` parameter to `parsec-close-request` instead, and not ask any question if this parameter is set.